CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,86 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Flatub is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible.
+As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
+
+This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
+
+We invite all those who participate in Flathub to help us create safe and positive experiences for everyone.
+
+## 2. Open Source Citizenship
+
+A supplemental goal of this Code of Conduct is to increase open source citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community.
+
+Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
```

Comment:
"abuses" -> "abuse" (well, depending on where "many forms" applies, but I think it reads better with inequality and abuse both in the singular)